docs/plans/2026-01-20-persistent-storage-design.md

```
@@ -2180,6 +2193,20 @@ $effect(() => {
 });
 ```
 
+**Note: Multi-Tab Behavior**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)